src/app/components/User/User.tsx

```diff
@@ -16,11 +16,11 @@ const User = ({ onClick }: UserProps): JSX.Element => {
   const { currentUser } = useCurrentUser();
 
   return (
-    <section
-      className={classes.container}
-      onClick={currentUser ? () => onClick(currentUser) : undefined}
-    >
-      <header className={classes.header}>
+    <section className={classes.container}>
+      <header
+        className={classes.header}
+        onClick={currentUser ? () => onClick(currentUser) : undefined}
```

Comment:
fyi: There is a problem with this `onClick` in my  pr #69. I changed the onClick property from `currentUser` to `currentUser.username` because the `onClick` sets the username in the `app.ts`. I also renamed it to `onUserClick`.